python/adjoint/wrapper.py

```diff
@@ -172,9 +167,12 @@ def _run_adjoint_simulation(self, monitor_values_grad):
                 'regions are present.')
         adjoint_sources = utils.create_adjoint_sources(self.monitors,
                                                        monitor_values_grad)
-        self.simulation.reset_meep()
+        # TODO refactor with optimization_problem.py #
+        self.simulation.restart_fields()
+        self.simulation.clear_dft_monitors()
         self.simulation.change_sources(adjoint_sources)
-        design_region_monitors = utils.install_design_region_monitors(
+        #                                            #
```

Comment:
Remove blank line?

Reply:
It's not a blank line. It's indicating what block of code corresponds to the TODO.